### PR TITLE
 Merging internal commits for release/8.0.1xx

### PR DESCRIPTION
**Pull request by marcpopMSFT**

### CODE REVIEW

**Comment by marcpopMSFT** [comment]
@layomia can you review and fix the [Microsoft.NET.Sdk.Web.ProjectSystem.targets](https://github.com/dotnet/sdk/pull/35407/files#diff-3f7b143775dcbfeec00acbb894d111d243a9aa038cad9245dbb960e50862d7a8) changes in this PR. This is meant to be an internal to public codeflow from RC1 but I believe it regressed some behavior because of that part of the change.

**Comment by marcpopMSFT** [comment]
@dotnet/sdk-container-builds-maintainers can you look at the src/Containers/Microsoft.NET.Build.Containers/AuthHandshakeMessageHandler.cs change?

File: src/Containers/Microsoft.NET.Build.Containers/AuthHandshakeMessageHandler.cs [hunk 1]
```diff
@@ -263,8 +263,10 @@ private async Task<(AuthenticationHeaderValue, DateTimeOffset)?> TryTokenGetAsyn
             var tokenResponse = await base.SendAsync(message, cancellationToken).ConfigureAwait(false);
             if (!tokenResponse.IsSuccessStatusCode)
             {
-                throw new UnableToAccessRepositoryException(_registryName);
+                await tokenResponse.LogHttpResponseAsync(_logger, cancellationToken).ConfigureAwait(false);
             }
+            tokenResponse.EnsureSuccessStatusCode();
+            _logger.LogTrace("Received '{statuscode}'.", tokenResponse.StatusCode);
```
Comment (baronfel) [inline src/Containers/Microsoft.NET.Build.Containers/AuthHandshakeMessageHandler.cs:269]: This change should be reverted - the more specific exception is used by other parts of the tooling to give users specific instructions/signal specific failure states.

**Comment by layomia** [comment]
> @layomia can you review and fix the [Microsoft.NET.Sdk.Web.ProjectSystem.targets](https://github.com/dotnet/sdk/pull/35407/files#diff-3f7b143775dcbfeec00acbb894d111d243a9aa038cad9245dbb960e50862d7a8) changes in this PR. This is meant to be an internal to public codeflow from RC1 but I believe it regressed some behavior because of that part of the change.

@marcpopMSFT taking a look.

**Comment by layomia** [comment]
Reinstating this commit by @captainsafia seems to fix the issue. https://github.com/dotnet/sdk/pull/35024.

**Review by baronfel — APPROVED** [review]
Containers looks good 👍

**Pull request merged**